### PR TITLE
Misc/create debug environment easily

### PR DESCRIPTION
**Pull request by Soichiro725**
### Purpose/goal of PR (Issue #)
Add docker environment to set up on local easily.

### CODE REVIEW

File: docker/HOWTO.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+## Stop
+```
+docker-compose rm -fs
+```
+
+## Use local html-swapper
+1. wovn.ini  
+Add the following to wovn.ini.sample
+```
+api_url = "http://host.docker.internal:3001/v0/"
```
Comment (a-bougouin) [inline docker/HOWTO.md]: This means we must add `host.docker.internal` to our `/etc/hosts` file, right?

Comment (Soichiro725) [reply docker/HOWTO.md]: `host.docker.internal` means host PC's `localhost`.
So,  `http://host.docker.internal:3001/v0/` means `localhost:3001/v0/`.
If you set `api_url = "localhost:3001/v0/"`, it means docker's localhost.

Comment (a-bougouin) [reply docker/HOWTO.md]: Ah I see, it is understood by docker as the localhost of the machine running the docker container?

Comment (Soichiro725) [reply docker/HOWTO.md]: Yes

Comment (a-bougouin) [reply docker/HOWTO.md]: Ok, thanks.

**Pull request merged**